sky/engine/core/dart/painting.dart

```diff
@@ -179,8 +180,13 @@ enum VertexMode {
   triangleFan,
 }
 
+/// An interface for recording drawing commands.
 class Canvas extends NativeFieldWrapperClass2 {
   void _constructor(PictureRecorder recorder, Rect bounds) native "Canvas_constructor";
+
+  /// Constructs an interface for recording drawing commands into the given picture recorder.
+  ///
+  /// Drawing commands outside the given bounds can be discarded by the implementation.
```

Comment:
This sentence is superficially helpful but when you start thinking about it really raises more questions than it answers. What is "outside"? Is the whole command discarded if any of it is outside, or just the parts that are outside? Is it reliably done or not? Are these bounds affected by transforms inside the canvas? How about outside the canvas; for example is the "real" clip axis aligned in some way? etc.